snowex_db/point_primary_variable_overrides.yaml

```diff
@@ -224,3 +225,26 @@ WIND_SPEED_10FT:
   map_from:
   - wsms_10ft_avg
   match_on_code: true
+CAMPAIGN:
+  auto_remap: true
+  code: campaign
+  description: Name of campaign site
+  map_from:
+  - location
+  match_on_code: true
+NAME:
```

Comment:
This entry is a bit confusing. Shouldn't this be 'PITID' as we do it for the metadata [variable in insitupy](https://github.com/jomey/insitupy/blob/4925e524655afaaf9bbab1cf8dbd43c654b4c1ad/insitupy/variables/basemetadatavariables.yaml#L21)